### PR TITLE
Add tooltips for Slider component

### PR DESCRIPTION
**Pull request by baomastr**
image

### CODE REVIEW

File: src/components/v2/Slider/index.stories.tsx [hunk 1]
```diff
@@ -9,19 +9,34 @@ export default {
...
+    onChange={console.log}
+    defaultValue={50}
+    step={5}
     mark={75}
     ariaLabel="Storybook slider"
     min={0}
     max={100}
+    trackTooltip="Storybook tooltip text for Track"
+    markTooltip="Storybook tooltip text for Mark"
+    isDisabled // @TODO: find a way to disable clicks
```
Comment (baomastr) [inline src/components/v2/Slider/index.stories.tsx]: got conflict with events.
have to disable clicks but leave hovers.

File: src/components/v2/Slider/index.tsx [hunk 2]
```diff
@@ -33,19 +41,53 @@ export const Slider = ({
...
+
+    return <span {...props} css={styles.mark} />;
+  };
+
+  const renderTrack = (props?: NonNullable<SliderTypeMap['props']['componentsProps']>['track']) => {
+    if (trackTooltip) {
+      return (
+        <div style={props?.style} css={[styles.trackWrapper, styles.hasTooltip]}>
+          <Tooltip title={trackTooltip}>
+            <Box {...props} style={{ width: '100%' }} />
```
Comment (therealemjy) [inline src/components/v2/Slider/index.tsx]: Let's not have inline styles

**Review by therealemjy — COMMENTED** [review]
Just one small comment. Looks good!

File: src/components/v2/Slider/index.tsx (hunk 2 above)
Comment (baomastr) [reply src/components/v2/Slider/index.tsx]: it has to be inlined in order to reassign inline styles. left a comment here.
image

Comment (baomastr) [reply src/components/v2/Slider/index.tsx]: passed undefined and left a comment

File: src/components/v2/Slider/index.stories.tsx (hunk 1 above)
Comment (baomastr) [reply src/components/v2/Slider/index.stories.tsx]: solved with passing value prop

File: src/components/v2/Slider/index.tsx [hunk 3]
```diff
@@ -1,20 +1,29 @@
...
   defaultValue: number;
   mark: number;
   step: number;
   ariaLabel: string;
   min: number;
   max: number;
   onChange: (value: number | number[]) => void;
-  isDisabled?: boolean;
+  trackTooltip?: string;
+  markTooltip?: string;
```
Comment (therealemjy) [inline src/components/v2/Slider/index.tsx]: Let's get rid of `defaultValue` then, since on our case the slider is never updated manually.

Comment (baomastr) [reply src/components/v2/Slider/index.tsx]: so, do you think we shouldn't have ability to change slider in future?

Comment (baomastr) [reply src/components/v2/Slider/index.tsx]: it should be just progressBar component then

Comment (therealemjy) [reply src/components/v2/Slider/index.tsx]: We don't have any slider in the designs, so yeah it is progress bar :)

Comment (baomastr) [reply src/components/v2/Slider/index.tsx]: done

**Review by therealemjy — APPROVED** [review]
Nice 👍

**Pull request merged**